__std__/cells/plutus-apps/packages/sphinx-build-readthedocs-site.nix

```diff
@@ -9,7 +9,11 @@ cell.library.pkgs.writeShellApplication {
     cell.packages.sphinx-toolchain
   ];
   text = ''
+    mkdir haddock
+    cp -aR ${cell.packages.combined-plutus-apps-haddock}/share/doc haddock
+    export PLUTUS_CORE_OBJECTS_INV=${cell.library.plutus-core-objects-inv}
     root="$(repo-root)"
-    sphinx-build -j 4 -n "$root/doc" "$root/doc/_build"
+    SPHINX_HADDOCK_DIR=$(readlink haddock) sphinx-build -W -j 4 -n "$root/doc" "$root/doc/_build"
```

Comment:
Yeah, I think I took `-W` out when I took out the haddock, since that causes lots of missing target warnings :(